changelog.md

```diff
@@ -1,3 +1,18 @@
+1.0.0
+===========
+### New Features:
+**Some constant values used in Microsoft Identity Web are available as public constants**. See feature request [#548](https://github.com/AzureAD/microsoft-identity-web/issues/548) for details.
```

Comment:
Now that we have an .md file, we might want to have hyphens to start each feature (for the moment this is all in one line :)

Reply:
ok. doesn't appear that way in VS code, but i'll fix it. mabye in another PR?